lib/config/rules/promise.js

```diff
@@ -9,7 +9,7 @@ module.exports = {
   // Avoid wrapping values in Promise.resolve or Promise.reject when not needed
   'promise/no-return-wrap': 'error',
   // Ensure that inside a then() you make sure to return a new promise or value.
-  'promise/always-return': 'error',
+  'promise/always-return': 'off',
```

Comment:
IMO it is not a bad practice to simply do a side effect-ey thing after a promise resolves. This rule also creates inconsistencies in what is allowed for promises versus what is allowed for async await (we do not require those functions to return a value, even though they are just promises under the hood)